cosinnus/static/less/v2/dashboard.less

```diff
@@ -547,26 +542,28 @@
 			line-height: 35px;
 			width: 100%;
 			white-space: nowrap;
-			color: @background-color;
-			background-color: @text-color-extra-pale;
+			color: @text-color;
```

Comment:
![timeline_attachment](https://github.com/user-attachments/assets/0c031183-4241-4c25-ab8a-7a857dc01776)

Der Wechsel der Farben in den Timeline Attachments war vorher zu kontrastarm, außerdem ist es schwierig, den Kontrast zu halten, wenn auch noch die Brandfarbe reinspielt, also habe ich es vereinfacht.